.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["es2015"]
+  "presets": ["es2015", "stage-0"]
```

Comment:
Sure ! I don't remember why I added it, I used `...spread` at some point I think.